### PR TITLE
Fix an error in comment

### PR DESCRIPTION
**Pull request by houjun41544**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: pkg/controller/volume/persistentvolume/scheduler_assume_cache.go [hunk 1]
```diff
@@ -378,7 +378,7 @@ type PVCAssumeCache interface {
 	AssumeCache
 
 	// GetPVC returns the PVC from the cache with the same
-	// namespace and the same name of the specified pod.
+	// namespace and the same name of the specified PVC.
```
Comment (lichuqiang) [inline pkg/controller/volume/persistentvolume/scheduler_assume_cache.go]: It's indeed a typo. But I'm wondering if the whole statement is not that appropriate here.
I think the comment here should be originally copied from [scheduler cache ](https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/internal/cache/interface.go#L84), which takes an object but not a key as input.

Maybe we should make it "GetPVC returns the PVC from the cache with given pvcKey" here.

**Comment by houjun41544** [comment]
@lichuqiang Could you please help to approve this ?

**Comment by houjun41544** [comment]
@msau42 would you please help to approve this ?

**Pull request merged**